css/properties/alignment-baseline.json

```
@@ -155,7 +155,7 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤80"
+                "version_added": "≤15"
```

Assuming this was supported from the beginning.

```suggestion
                "version_added": "1"
```